src/vector/jitsi/index.scss

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright 2020 New Vector Ltd.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// TODO: Match the user's theme instead of hardcoding a bunch of colors
+
+@font-face {
+    font-family: 'Nunito';
+    font-style: normal;
+    font-weight: 400;
+    src: url('~matrix-react-sdk/res/fonts/Nunito/Nunito-Regular.ttf') format('truetype');
+}
+
+body {
+    font-family: Nunito, Arial, Helvetica, sans-serif;
+    background-color: #181b21;
+    color: #edf3ff;
+}
+
+body, html {
+    padding: 0;
+    margin: 0;
+}
+
+#jitsiContainer {
+    position: fixed;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+}
+
+#joinButtonContainer {
+    display: table;
+    position: absolute;
+    height: 100%;
+    width: 100%;
+}
+
+.join-conference-boat {
+    display: table-cell;
+    vertical-align: middle;
+}
+
+.join-conference-prompt {
```

Comment:
Seems like a mix of `#camelCase` and `.hyphen-case` naming in here, maybe better to pick one?